src/gobstuf/rest/brp/base_view.py

```diff
@@ -21,13 +21,16 @@ def headers_required_decorator(headers):
     :param headers:
     :return:
     """
+
     def headers_required(f):
         def decorator(*args, **kwargs):
             if not all([request.headers.get(h) for h in headers]):
                 return abort(Response(response='Missing required MKS headers', status=400))
 
             return f(*args, **kwargs)
+
         return decorator
+
     return headers_required
```

Comment:
method has been changed without reason?

Reply:
Think this was some kind of autoformating. Reverted.